### DISCUSSION TITLE
The iteration numbers of ART/SART/SIRT

### DISCUSSION THREAD

**Discussion by AGan2017** [astra-toolbox/astra-toolbox#499] [Q&A]
Hi everyone, I have an question about the iteration numbers of SART algorithm, could you explain for me? thank you very much.
the iteration numbers of SART must be the multiple of the projection number? like the example of the "SART Algorithm" (https://astra-toolbox.com/docs/algs/SART.html), the iteration number is set to 10*180, how can I get the result of one iteration of SART ? (set to 1 is error ), I want to run a iterative procedure, and get the result of each iteration.

**Comment by askorikov** [comment]
Hi! I'm not completely sure what you mean, but I don't think the number of iterations for SART needs to be a multiple of the number of projections. Also, running 1 iteration doesn't seem to give an error for me (with astra-toolbox 2.2.0).
